.travis.yml

```diff
@@ -36,3 +24,63 @@ cache:
   - "$HOME/.coursier/cache"
   - "$HOME/.ivy2/cache"
   - "$HOME/.sbt/boot"
+
+jobs:
+  include:
+    - &tests
+      stage: Test
+      scala: 2.12.2-bin-typelevel-4
+      env: SCALAZ_VERSION=7.2.13
+      script: $TRAVIS_BUILD_DIR/bin/specialTest.sh
+
+    - <<: *tests
+      scala: 2.12.2-bin-typelevel-4
+      env: SCALAZ_VERSION=7.1.13
+      script: $TRAVIS_BUILD_DIR/bin/test.sh
+    - <<: *tests
+      scala: 2.11.11-bin-typelevel-4
+      env: SCALAZ_VERSION=7.2.13
+      script: $TRAVIS_BUILD_DIR/bin/test.sh
+    - <<: *tests
+      scala: 2.11.11-bin-typelevel-4
+      env: SCALAZ_VERSION=7.1.13
+      script: $TRAVIS_BUILD_DIR/bin/test.sh
+    - <<: *tests
+      scala: 2.10.6
+      env: SCALAZ_VERSION=7.2.13
+      script: $TRAVIS_BUILD_DIR/bin/test.sh
+    - <<: *tests
+      scala: 2.10.6
+      env: SCALAZ_VERSION=7.1.13
+      script: $TRAVIS_BUILD_DIR/bin/test.sh
+
+    - stage: Publish Site
+      scala: 2.12.2-bin-typelevel-4
+      env: SCALAZ_VERSION=7.2.13
+      script: $TRAVIS_BUILD_DIR/bin/publishSite.sh
+
+    - &publishSonatype
```

Comment:
I'm a little wary because this added 60 lines of code with repeated version strings to accomplish the same result.  But when the matrix gets simplified in 0.17, I suppose the pendulum swings a bit?